src/sentry/replays/lib/query.py

```diff
@@ -358,13 +362,45 @@ def _filter_tag_by_value(
         return Condition(
             Function(
                 function,
-                parameters=[all_values_for_tag_key(key, Column("tk"), Column("tv")), values],
+                parameters=[
+                    all_values_for_tag_key(
+                        key, Column(self.tag_key_alias), Column(self.tag_value_alias)
+                    ),
+                    values,
+                ],
             ),
             Op.EQ,
             expected,
         )
 
 
+class InvalidField(Field):
+    _operators = [Op.EQ, Op.NEQ, Op.IN, Op.NOT_IN]
+    _python_type = str
+
+    def as_condition(
+        self, _: str, operator: Op, value: Union[List[str], str], is_wildcard: bool = False
+    ) -> Condition:
+        raise ParseError()
+
+    def _wildcard_condition(self, operator: Op, value: str):
+        raise ParseError()
+
+    def _has_condition(
+        self,
+        operator: Op,
+        value: Union[List[str], str],
+    ) -> Condition:
+        raise ParseError()
+
+    def _has_any_condition(
+        self,
+        operator: Op,
+        values: Union[List[str], str],
+    ) -> Condition:
+        raise ParseError()
```

Comment:
I know in our use case we catch these exceptions and our aggregated schema does not contain these InvalidField types.  However, since this is a library class I think we should return an error message.  Even if its just "Invalid field specified".